src/helpers/em.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+/** @module Helpers */
+
+import stripUnit from './stripUnit'
+
+/**
+ * Convert pixel value to ems. The default base value is 16px, but can be changed by passing a
+ * second argument to the function.
+ * @static
+ * @param {string|number} pxval - The pixel value you want to convert
+ * @param {string|number} [base = "16px"] - The base size to convert from
+ * @return {String} The converted value
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   'height': em('16px')
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   height: ${em('16px')}
+ * `
+ *
+ * // CSS in JS Output
+ *
+ * element {
+ *   'height': '1em'
+ * }
+ */
+
+/* eslint-disable no-param-reassign */
+function em(pxval: string|number, base?: string|number = '16px') {
+  if (typeof pxval === 'string') {
+    pxval = stripUnit(pxval)
+  }
+
+  if (typeof base === 'string') {
+    base = stripUnit(base)
+  }
+
+  if (typeof pxval === 'string') {
+    throw new Error(`Passed invalid pixel value ("${pxval}") to em(), please pass a value like "12px" or 12.`)
```

Comment:
Should we handle the case where a valid string value is passed but of a unit that is not px. For example if someone tried to do `em(12rem)`. Since the method name is just `em` it isn't clear immediately (outside of docs) that states it only accepts `px`.